acts

```diff
@@ -149,7 +149,7 @@ archives=$($tarsnap --list-archives | sort -n)
 # PART 4: Trim old backups.
 
 if [ "$backuprc" != "0" ] ; then
-    log_message "One of the backups failed -- not deleting old backups"
+    log_verbose "at=error type=$archivetype"
```

Comment:
I'm wondering if `error` is the right level for this line, it is mostly added to aid people read logfiles (since it just ensures any error is referred to be two lines). What do you think?